src/Compilers/CSharp/Test/Emit3/FieldKeywordTests.cs

```diff
@@ -0,0 +1,344 @@
+                {
+                    public object P => Initialize(out field, 1);
+                    public object Q { get => Initialize(out field, 2); }
+                    static object Initialize(out object field, object value)
+                    {
+                        field = value;
+                        return field;
+                    }
+                }
+                class Program
+                {
+                    static void Main()
+                    {
+                        var c = new C();
+                        Console.WriteLine((c.P, c.Q));
+                    }
+                }
+                """;
+            var verifier = CompileAndVerify(source, expectedOutput: "(1, 2)");
+            verifier.VerifyIL("C.P.get", """
+                {
+                  // Code size       18 (0x12)
+                  .maxstack  2
+                  IL_0000:  ldarg.0
+                  IL_0001:  ldflda     "object C.<P>k__BackingField"
+                  IL_0006:  ldc.i4.1
+                  IL_0007:  box        "int"
+                  IL_000c:  call       "object C.Initialize(out object, object)"
+                  IL_0011:  ret
+                }
+                """);
+            verifier.VerifyIL("C.Q.get", """
+                {
+                  // Code size       18 (0x12)
+                  .maxstack  2
+                  IL_0000:  ldarg.0
+                  IL_0001:  ldflda     "object C.<Q>k__BackingField"
+                  IL_0006:  ldc.i4.2
+                  IL_0007:  box        "int"
+                  IL_000c:  call       "object C.Initialize(out object, object)"
+                  IL_0011:  ret
+                }
+                """);
+        }
+
+        [Fact]
+        public void Field_03()
+        {
+            string source = """
+                using System;
+                using System.Reflection;
+                class C
+                {
+                    public object P { get { return field; } init { field = 1; } }
+                    public object Q { init { field = 2; } }
+                }
+                class Program
+                {
+                    static void Main()
+                    {
+                        foreach (var field in typeof(C).GetFields(BindingFlags.NonPublic | BindingFlags.Instance))
+                            Console.WriteLine("{0}: {1}", field.Name, field.IsInitOnly);
+                    }
+                }
+                """;
+            CompileAndVerify(source, targetFramework: TargetFramework.Net80, verify: Verification.Skipped, expectedOutput: IncludeExpectedOutput("""
+                <P>k__BackingField: False
+                <Q>k__BackingField: False
+                """));
+        }
+
+        [Fact]
+        public void FieldReference_01()
+        {
+            string source = """
+                class C
+                {
+                    static C _other = new();
+                    object P => _other.field;
```

Comment:
`C` doesn't have any backing fields here; consider adding a setter that does use `field` to ensure that it's actually generated.